mitodl/ocw-studio
# update ckeditor to v35

#### Pre-Flight checklist

  - [x] Code is tested **Somewhat. We don't really have any tests that fully interact with CKEditor since it uses browser-only APIs.**
  - [x] Changes have been manually teste

#### What are the relevant tickets?
Closes #1511, progress toward #1061 

#### What's this PR do?
This PR updates CKEditor to version 35.4.0

#### How should this be manually tested?
1. On this branch, restart docker so packages are installed afresh
2. Check that rich-text editing still works.

#### Any background context you want to provide?
Our immediate motivation for updating CKEditor is that v31 seemed to have a strange interaction between subscript/superscript and link plugins. See https://github.com/mitodl/ocw-studio/issues/1061#issuecomment-1349633748
